plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/impl/api/rename.kt

```diff
@@ -28,55 +31,90 @@ class RenameInto : AbstractSchemaModificationInterpreter() {
         val columns = receiver.columns.resolve(receiver.schema)
         require(columns.size == newNames.size)
         var i = 0
-        return receiver.schema.map(columns.mapTo(mutableSetOf()) { it.path.path }, nextName = { newNames[i].also { i += 1 } })
+        return receiver.schema.map(
+            selected = columns.mapTo(mutableSetOf()) { it.path.path },
+            nextName = { newNames[i].also { i += 1 } },
+        )
     }
 }
 
-internal fun PluginDataFrameSchema.map(selected: ColumnsSet, nextName: () -> String): PluginDataFrameSchema {
-    return PluginDataFrameSchema(
-        f(columns(), nextName, selected, emptyList())
+internal fun PluginDataFrameSchema.map(selected: ColumnsSet, nextName: () -> String): PluginDataFrameSchema =
+    PluginDataFrameSchema(
+        f(columns(), nextName, selected, emptyList()),
     )
-}
 
-internal fun f(columns: List<SimpleCol>, nextName: () -> String, selected: ColumnsSet, path: List<String>): List<SimpleCol> {
-    return columns.map {
+internal fun f(
+    columns: List<SimpleCol>,
+    nextName: () -> String,
+    selected: ColumnsSet,
+    path: List<String>,
+): List<SimpleCol> =
+    columns.map {
         val fullPath = path + listOf(it.name)
         when (it) {
             is SimpleColumnGroup -> {
                 val group = if (fullPath in selected) {
                     it.rename(nextName())
-                }  else {
+                } else {
                     it
                 }
                 group.map(selected, fullPath, nextName)
             }
+
             is SimpleFrameColumn -> {
                 val frame = if (fullPath in selected) {
                     it.rename(nextName())
-                }  else {
+                } else {
                     it
                 }
                 frame.map(selected, fullPath, nextName)
             }
+
             is SimpleDataColumn -> if (fullPath in selected) {
                 it.rename(nextName())
             } else {
                 it
             }
         }
     }
-}
 
-internal fun SimpleColumnGroup.map(selected: ColumnsSet, path: List<String>, nextName: () -> String): SimpleColumnGroup {
-    return SimpleColumnGroup(
-        name,
-        f(columns(), nextName, selected, path)
+internal fun SimpleColumnGroup.map(
+    selected: ColumnsSet,
+    path: List<String>,
+    nextName: () -> String,
+): SimpleColumnGroup =
+    SimpleColumnGroup(
+        name = name,
+        columns = f(columns(), nextName, selected, path),
     )
-}
 
-internal fun SimpleFrameColumn.map(selected: ColumnsSet, path: List<String>, nextName: () -> String): SimpleFrameColumn {
-    return SimpleFrameColumn(
-        name,
-        f(columns(), nextName, selected, path)
+internal fun SimpleFrameColumn.map(
```

Comment:
Please, for easy and fast reviews let's hold changes related to linting and other non-functional changes

Reply:
I put it in a separate commit, so you can skip those if you wish